lib/css/components/badges.less

```diff
@@ -174,6 +183,44 @@
     .highcontrast-mode({ border-color: currentColor; });
 }
 
+//  $$  States
+//  ---------------------------------------------------------------------------
+&.s-badge__danger {
+    color: var(--red-900);
+    background-color: var(--red-100);
+    border-color: var(--red-600);
+
+    &.s-badge__filled {
```

Comment:
Do we want to add filled versions to all badges? As of right now, the api is a tad confusing since the filled modifier doesn't work on all types. Alternatively, this suggestion may fall into YAGNI territory.

Reply:
Also, since these are all already "filled' (unlike s-btn), should we perhaps follow the convention set by s-notice and use "important" instead?

Reply:
I feel you on this. `s-badge__important` is already taken (see [badges/#number-counts](https://deploy-preview-914--stacks.netlify.app/product/components/badges/#number-counts) though. `__filled` was the best option I could come up with but I'm not totally happy with it. I'll see if I can come up with anything better.

Reply:
Ah shoot, I forgot about that one. Good call. Let me think on this one a bit as well.

Reply:
Let's stick with "filled" for now. We can switch over to "important" as a breaking change once we have a few more under our belt.